go.work.sum

```diff
@@ -165,6 +165,7 @@ github.com/coreos/bbolt v1.3.2 h1:wZwiHHUieZCquLkDL0B8UhzreNWsPHooDAG3q34zk0s=
 github.com/coreos/etcd v3.3.13+incompatible h1:8F3hqu9fGYLBifCmRCJsicFqDx/D68Rt3q1JMazcgBQ=
 github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e h1:Wf6HqHfScWJN9/ZjdUKyjop4mf3Qdd+1TvvltAvM3m8=
 github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f h1:lBNOc5arjvs8E5mO2tbpBpLoyyu8B6e44T7hJy6potg=
+github.com/cpuguy83/go-md2man/v2 v2.0.4 h1:wfIWP927BUkWJb2NmU/kNDYIBTh/ziUX91+lVfRxZq4=
```

Comment:
Why are we adding a new dependency?

Reply:
I did not add it, this is a result of running go mod tidy after updating the cobra requirement.

Reply:
Correction - results of running "go work vendor" - it also removed the OWNERS file. Maybe there is a better way to do this?

Reply:
ok, I just pulled again, and I see that current code is already using cobra 1.8.1, added in 52e4fbdd85bae16891c05235c22ad01f52f82bd4

I'll update the pr to remove the first 2 commit, they seems to be unneeded.

Reply:
The unneeded commit was removed.